avaframe/tests/data/avaFilterTest/com1DFA/configurationFiles/relGar_ent_dfa_789ce37489.ini

```diff
@@ -0,0 +1,119 @@
+[GENERAL]
+simTypeList = ent
+modelType = dfa
+resType = ppr|pfd|pfv
+tSteps = 1
+initPartDistType = random
+initialiseParticlesFromFile = False
+particleFile = 
+seed = 12345
+iniStep = False
+subgridMixingFactorIni = 10.
+dtIni = 0.1
+sphOptionIni = 2
+stopCritIni = 0.1
+stopCritIniSmall = 1.001
+maxIterations = 100
+bufferZoneFactor = 4
+additionallyFixedFactor = 0.275
```

Comment:
Didn't we say we only put parameters that change from the default cfg in here?

Reply:
but this is not an input for running com1DFA this is used to test the filtering  functions that read the ini files from the given input folder or from avaDir/Outputs/com1DFA/configurationFiles so here i need the full ini